src/bundle/Controller/ContentViewController.php

```diff
@@ -525,6 +528,45 @@ static function (Repository $repository) use ($contentInfo): bool {
 
         $view->addParameters(['content_has_reverse_relations' => $hasReverseRelations]);
     }
+
+    /**
+     * @param \Ibexa\Core\MVC\Symfony\View\ContentView $view
+     */
+    private function supplyUserInvitation(ContentView $view): void
+    {
+        $content = $view->getContent();
+        $contentType = $this->contentTypeService->loadContentType(
+            $view->getContent()->contentInfo->contentTypeId,
```

Comment:
Could you use here strict getter instead of magic due to performance?
```suggestion
            $view->getContent()->getVersionInfo()->getContentInfo()->contentTypeId,
```